### PR TITLE
Initial governance and README docs for the repository

### PR DESCRIPTION
**Pull request by swcurran**
Signed-off-by: Stephen Curran <swcurran@gmail.com>

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,29 +1,44 @@
...
-- Include unit and integration tests for any new features and updates to existing tests
+- Include unit and integration tests for any new features and updates to
+  existing tests
 - Ensure that the unit and integration tests run successfully.
 - Check that the lint tests pass
 
-## Important
+### Rebase
+
 Use `git rebase origin/master` to limit creating merge commits
```
Comment (berendsliedrecht) [inline CONTRIBUTING.md]: This should be `origin/main` I believe.

**Review by berendsliedrecht — APPROVED** [review]

**Comment by swcurran** [comment]
@blu3beri -- added the setup.py and bls.py -- could you check those out?  Do they look right to you?

Let me know if other adjustments are needed...

**Comment by swcurran** [comment]
@andrewwhitehead -- FYI -- proposing name to be "indy-bls" (from "ursa") and using version `0.1.1`.

File: setup.py [hunk 2]
```diff
@@ -5,21 +5,21 @@
     long_description = fh.read()
 
 setup(
-    name='python-ursa',
-    version='0.1.0',
+    name='python-indy-bls',
```
Comment (berendsliedrecht) [inline setup.py]: Not entirely too sure, but could the name just be "indy-bls"? Might be a bit too verbose if the package has a prefix of python on the registry.

**Review by berendsliedrecht — APPROVED** [review]

File: setup.py (hunk 2 above)
Comment (swcurran) [reply setup.py]: Good catch -- I fixed it in one place not the other.  #rookie

**Pull request merged**